src/controls/adaptiveCardHost/fluentUI/Elements.tsx

```diff
@@ -0,0 +1,838 @@
+        let theme = this.hostConfig.hostCapabilities.getCustomProperty(hostCapabilitiesFluentUIThemeKey);
+
+        return <IconButton
+            default={true}
+            text={this.inlineAction.title}
+            ariaDescription={this.inlineAction.title}
+            className={this.hostConfig.makeCssClassName("ac-inlineActionButton", "iconOnly")}
+            theme={theme}
+            styles={{
+                icon: {
+                    height: `100%`,
+                },
+                root: inlineButtonRootStyle,
+            }}
+            iconProps={{
+                imageProps: {
+                    height: "100%",
+                    src: this.inlineAction.iconUrl,
+                },
+            }}
+            onClick={this.inlineActionClickHandler}
+        />;
+    }
+
+    private buildTextOnlyInlineActionActionButton = (): JSX.Element => {
+        let theme = this.hostConfig.hostCapabilities.getCustomProperty(hostCapabilitiesFluentUIThemeKey);
+
+        return <DefaultButton
+            className={this.hostConfig.makeCssClassName("ac-inlineActionButton", "textOnly")}
+            text={this.inlineAction.title}
+            ariaDescription={this.inlineAction.title}
+            onClick={this.inlineActionClickHandler}
+            styles={{
+                root: inlineButtonRootStyle,
+            }}
+            theme={theme}
+        />;
+    }
+
+    public getJsonTypeName(): string {
+        return "Input.Text";
+    }
+
+    public focus() {
+        if (this.refControl)
+            this.refControl.focus();
+    }
+
+    protected updateInputControlAriaLabelledBy() {
+        if (this.element) {
+            let joinedLabelIds = this.getAllLabelIds().join(" ");
+
+            if (joinedLabelIds) {
+                this.element.setAttribute("aria-labelledby", joinedLabelIds);
+            } else {
+                this.element.removeAttribute("aria-labelledby");
+            }
+        }
+    }
+
+    public getActionById(id: string) {
+        let result = super.getActionById(id);
+
+        if (!result && this.inlineAction) {
+            result = this.inlineAction.getActionById(id);
+        }
+
+        return result;
+    }
+
+    public isSet(): boolean {
+        return this.value ? true : false;
+    }
+
+    public isValid(): boolean {
+        if (!this.value) {
+            return !this.isRequired;
+        }
+
+        if (this.regex) {
+            return new RegExp(this.regex, "g").test(this.value);
+        }
+
+        return true;
+    }
+
+    public get value(): string | undefined {
+        if (this.renderedInputControlElement) {
+            return this.refControl.value;
+        }
+        else {
+            return undefined;
+        }
+    }
+}
+
+export class FluentUITimeInput extends Input {
+    private static convertTimeStringToDate(timeString: string): Date {
+        return new Date("1973-09-04T" + timeString + ":00Z");
```

Comment:
Is this a hardcoded date from testing?

Reply:
Yes, it's for comparing time only.
It's the same code present into the default implementation: https://github.com/microsoft/AdaptiveCards/blob/f764013a76899b6776291edd1de1cbe1506e7faf/source/nodejs/adaptivecards/src/card-elements.ts#L4032